README.md

```diff
@@ -68,16 +68,28 @@ Example:
 
 # Develop
 
-Setup fixtures for bench test
+Build binaries
 
 ```
-git submodule update --init
+make
 ```
 
-Build binaries
+For Mac M1 users, there are a couple of extra steps before building
 
+- Patch binding.gyp in the local repo clone to comment out linking instructions as follows
 ```
-make
+#  'make_global_settings': [
+#    ['CXX', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
+#    ['CC', '<(module_root_dir)/mason_packages/.link/bin/clang'],
+#    ['LINK', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
+#    ['AR', '<(module_root_dir)/mason_packages/.link/bin/llvm-ar'],
+#    ['NM', '<(module_root_dir)/mason_packages/.link/bin/llvm-nm']
+#  ],
+```
```

Comment:
Not sure if this is too verbose here, but GH supports diffs in markdown, which is pretty neat:

```diff
-  'make_global_settings': [
-    ['CXX', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
-    ['CC', '<(module_root_dir)/mason_packages/.link/bin/clang'],
-    ['LINK', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
-    ['AR', '<(module_root_dir)/mason_packages/.link/bin/llvm-ar'],
-    ['NM', '<(module_root_dir)/mason_packages/.link/bin/llvm-nm']
-  ],
+  # 'make_global_settings': [
+  #   ['CXX', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
+  #   ['CC', '<(module_root_dir)/mason_packages/.link/bin/clang'],
+  #   ['LINK', '<(module_root_dir)/mason_packages/.link/bin/clang++'],
+  #   ['AR', '<(module_root_dir)/mason_packages/.link/bin/llvm-ar'],
+  #   ['NM', '<(module_root_dir)/mason_packages/.link/bin/llvm-nm']
+  # ],
```

Reply:
yeah, it's nice but it makes it a little verbose for a generic markdown editor like vi or vscode where I usually read Readme's, so I kept it concise.